extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/cors/CORSFilter.java

```diff
@@ -121,4 +119,13 @@ public void handle(RoutingContext event) {
             }
         }
     }
-}
+
+    private boolean containsOnlyWildcard(Optional<List<String>> optionalList) {
```

Comment:
do you have a better label than `containsOnlyWildcard` for this method?

maybe something like `isConfiguredWithWildcard`

Reply:
Maybe you could make this static and add a small test making sure it responds correctly?

Reply:
> Maybe you could make this static and add a small test making sure it responds correctly?

Actually, it is a good one. This should have been tested indirectly in this `CORSWildcardSecurityTestCase` but now that I've seen this suggestion I've realized `CORSWildcardSecurityTestCase` needs its application properties updated. 
@xumk please update it to check both wildcard origins and headers